cloud-platform/app-deploy-helm.md

```diff
@@ -0,0 +1,151 @@
+
+### Requirements
+It is assumed you have the following: 
+
+ - You have a basic understanding of what [Kubernetes](https://kubernetes.io/) is.
+ - You have [created an environment for your application](/cloud-platform/env-create)
+ - [Kubectl](https://kubernetes.io/docs/tasks/tools/install-kubectl/) is installed on your local machine.
+ - [Docker](https://docs.docker.com/install/) is installed on your local machine. 
+ - [Authentication](/cloud-platform/authenticate) has been established with the Non-production cluster.
+
+## Deploy the app
+The reference application we're going to use is a very simple Django application with an on-cluster Postgresql database.
+
+> Note: Even though we are going to install a database within the Kubernetes cluster, it is recommended to use a database as a service offering such as [AWS RDS](https://aws.amazon.com/rds/) if running in production. 
+
+Lucky for you the Helm deployment manifests have been pre-written for this exercise. But if you wish to know more about these files and what they do have a quick browse of the [README](https://github.com/ministryofjustice/cloud-platform-reference-app/tree/master/helm_deploy/django-app/README.md).
+
+### Set up
+First and foremost we need to clone our reference application and subsequently change directory: 
+
+    $ git clone https://github.com/ministryofjustice/cloud-platform-reference-app.git
+    $ cd cloud-platform-reference-app
+
+You now have a functioning git repository that contains a Django simple application. Have a browse around and get familiar with the directory structure.
+
+### Nothing up my sleeves!
+Let's make use of the command line tool `kubectl` to browse around the cluster to see what it looks like before we deploy our application: 
+
+    $ kubectl get pods -namespace <env-name>
+*The `<env-name>` here is the environment you created, listed in the requirements section at the beginning of the document.* 
+
+You should be met with a simple response of: 
+
+    No resources found.
+
+If you receive the below error message then you've either not typed in your environment name correctly or you don't have permission to perform a `get pods` command. Either way, you'll need to go back and review the [Creating an Environment](https://ministryofjustice.github.io/cloud-platform-user-docs/cloud-platform/env-create/#creating-a-cloud-platform-environment) document previously mentioned. 
+
+    $ Error from server (Forbidden): pods is forbidden: User "test-user" cannot list pods in the namespace "demo"
+
+### Using Helm
+First, what is Helm? Helm allows you to manage application deployment to Kubernetes using Charts. Have a quick read of some of the many features of Helm Charts [here](https://docs.helm.sh/developing_charts/). 
```

Comment:
Around here is I think where it is best to put the install Helm bit. It might be worth saying that Helm connects to an instance of Tiller running in your namespace (it gets installed as part of environment creation).